Lib/sqlite3/_completer.py

```diff
@@ -1,15 +1,41 @@
-from _sqlite3 import SQLITE_KEYWORDS
 from contextlib import contextmanager
 
-_completion_matches = []
 
+KEYWORDS = ("ABORT", "ACTION", "ADD", "AFTER", "ALL", "ALTER", "ALWAYS",
+            "ANALYZE", "AND", "AS", "ASC", "ATTACH", "AUTOINCREMENT",
+            "BEFORE", "BEGIN", "BETWEEN", "BY", "CASCADE", "CASE", "CAST",
+            "CHECK", "COLLATE", "COLUMN", "COMMIT", "CONFLICT",
+            "CONSTRAINT", "CREATE", "CROSS", "CURRENT", "CURRENT_DATE",
+            "CURRENT_TIME", "CURRENT_TIMESTAMP", "DATABASE", "DEFAULT",
+            "DEFERRABLE", "DEFERRED", "DELETE", "DESC", "DETACH",
+            "DISTINCT", "DO", "DROP", "EACH", "ELSE", "END", "ESCAPE",
+            "EXCEPT", "EXCLUDE", "EXCLUSIVE", "EXISTS", "EXPLAIN", "FAIL",
+            "FILTER", "FIRST", "FOLLOWING", "FOR", "FOREIGN", "FROM",
+            "FULL", "GENERATED", "GLOB", "GROUP", "GROUPS", "HAVING", "IF",
+            "IGNORE", "IMMEDIATE", "IN", "INDEX", "INDEXED", "INITIALLY",
+            "INNER", "INSERT", "INSTEAD", "INTERSECT", "INTO", "IS",
+            "ISNULL", "JOIN", "KEY", "LAST", "LEFT", "LIKE", "LIMIT",
+            "MATCH", "MATERIALIZED", "NATURAL", "NO", "NOT", "NOTHING",
+            "NOTNULL", "NULL", "NULLS", "OF", "OFFSET", "ON", "OR",
+            "ORDER", "OTHERS", "OUTER", "OVER", "PARTITION", "PLAN",
+            "PRAGMA", "PRECEDING", "PRIMARY", "QUERY", "RAISE", "RANGE",
+            "RECURSIVE", "REFERENCES", "REGEXP", "REINDEX", "RELEASE",
+            "RENAME", "REPLACE", "RESTRICT", "RETURNING", "RIGHT",
+            "ROLLBACK", "ROW", "ROWS", "SAVEPOINT", "SELECT", "SET",
+            "TABLE", "TEMP", "TEMPORARY", "THEN", "TIES", "TO",
+            "TRANSACTION", "TRIGGER", "UNBOUNDED", "UNION", "UNIQUE",
+            "UPDATE", "USING", "VACUUM", "VALUES", "VIEW", "VIRTUAL",
```

Comment:
According to the [docs](https://sqlite.org/c3ref/keyword_check.html), `VACUUM` might not be a keyword on some builds of SQLite.
This should ask SQLite for the list.

Since you mentioned you're not comfortable with C, I took a stab at it; see https://github.com/tanloong/cpython/pull/2